src/org/robockets/steamworks/OI.java

```diff
@@ -1,21 +1,115 @@
 package org.robockets.steamworks;
 
-import org.robockets.steamworks.drivetrain.ToggleDriveMode;
-
 import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj.buttons.Button;
 import edu.wpi.first.wpilibj.buttons.JoystickButton;
 
+import org.robockets.commons.RelativeDirection;
+import org.robockets.steamworks.ballintake.IntakeBalls;
+import org.robockets.steamworks.ballintake.SpinBallIntakeRollers;
+import org.robockets.steamworks.climber.Climb;
+import org.robockets.steamworks.commands.KillEverything;
+import org.robockets.steamworks.commands.MakeExtraSpace;
+import org.robockets.steamworks.commands.MoveConveyor;
+import org.robockets.steamworks.commands.MoveElevator;
+import org.robockets.steamworks.commands.Shoot;
+import org.robockets.steamworks.commands.SpinSpinners;
+import org.robockets.steamworks.intakeflap.IntakeFlap;
+import org.robockets.steamworks.intakeflap.IntakeToPos;
+import org.robockets.steamworks.intakeflap.MoveIntakeFlap;
+
 /**
  * This class is the glue that binds the controls on the physical operator
  * interface to the commands and command groups that allow control of the robot.
  */
 public class OI {
-    public static Joystick joystick = new Joystick(0);
+    public static Joystick joystick = new Joystick(0); // XBox Controller
+    public static Joystick attack3Right = new Joystick(1);
+    public static Joystick attack3Left = new Joystick(2);
+
+    public static Joystick launchpad = new Joystick(3);
```

Comment:
The first `Joystick` (not shown) has a comment indicating that it's the Xbox controller. I'd like comments for the other three `Joystick`s.